Yosemite/YosemiteTests/Stores/ProductStoreTests.swift

```diff
@@ -737,7 +743,7 @@ final class ProductStoreTests: XCTestCase {
     ///
     func testUpdateStoredProductEffectivelyPersistsNewProduct() {
         let productStore = ProductStore(dispatcher: dispatcher, storageManager: storageManager, network: network)
-        let remoteProduct = sampleProduct()
+        let remoteProduct = sampleProduct(downloadable: true)
```

Comment:
I'm curious about this change: why `true`? 💭 It seems that it's not used.

Reply:
Actually it has been used here.
That `remoteProduct ` will have 3 downloadable files since it's `sampleProduct(downloadable: true)`
So at line 755, we are checking that there are not downloadable files before `upsertStoredProduct`

And after `upsertStoredProduct` the `remoteProduct` we should have 3 `ProductDownload` which has been checked at line 767